xbmc/profiles/dialogs/GUIDialogProfileSettings.cpp

```diff
@@ -150,7 +150,8 @@ bool CGUIDialogProfileSettings::ShowForProfile(unsigned int iProfile, bool first
         // copy masterprofile guisettings to new profile guisettings
         // If the user selects 'start fresh', do nothing as a fresh
         // guisettings.xml will be created on first profile use.
-        if (CGUIDialogYesNo::ShowAndGetInput(20058,20048,20102,20022,20044,20064))
+        bool cancelled = false;
+        if (CGUIDialogYesNo::ShowAndGetInput(20058, 20048, cancelled, 20044, 20064))
```

Comment:
I'm sorry, I missed something when i check this yesterday. We can't use the overloaded method I used here. So basically stick to what it was before (without cancelled) and only replace 20102 and 20022 with -1.